rfcs/NNNN-hot-elements.md

```diff
@@ -0,0 +1,77 @@
+---
+Status: Active
+Champions: {{ rictic }}
+PR: {{ 15 }}
+---
+
+# Hot Elements
+
+A new Lit Labs package for supporting hot module replacement with custom elements.
+
+## Objective
+
+### Goals
```

Comment:
Which HMR protocol are we supporting? `import.meta.hot`?

Reply:
Yes, see a few lines down:

> - Support @web/dev-server-hmr specifically and the [esm-hmr](https://github.com/FredKSchott/esm-hmr) protocol more generally.